### PR TITLE
Add overview of Labs program format

### PR DESCRIPTION
**Pull request by MaxWofford**
This closes hackedu/meta#141

### CODE REVIEW

File: meta/labs/README.md [hunk 1]
```diff
@@ -3,6 +3,24 @@
 This document will contain a brief overview of our labs program and what it'll
 entail going into the 2015 fall semester.
 
+## Format
+
+The Labs program will follow this format:
+
+1. Workshops
+   - These are based on the workshops from Hack Camp
```
Comment (jonleung) [inline meta/labs/README.md]: Clubs may create their own workshops as well : )

Comment (jonleung) [inline meta/labs/README.md]: I know that we didn't discuss the above ^, but club leaders are asking me if they can create their own workshops, someone proposed an Agar.io one and I thought that sounded awesome!

Comment (jonleung) [inline meta/labs/README.md]: This top section (not including the case studies) I don't think accurately captures labs.

Comment (jonleung) [inline meta/labs/README.md]: I would include what the objective of what Labs is and leave it at that. Perhaps link to the workshops as something that clubs can use but not that they are mandatory.

**Comment by zachlatta** [comment]
I decided to go ahead and get this into master. @jonleung: can you submit a PR to add anything you'd like to?

Cherry picked into master in e33d8b4.

**Pull request closed without merging**